packages-common/pluggable-widgets-tools/tests/commands.spec.js

```diff
@@ -14,9 +14,9 @@ const CONFIGS = [
     ["web", "js", "8.3"],
     ["native", "ts", "8.6"],
     ["web", "ts", "8.6"],
-    ["web", "js", "8.7"],
-    ["web", "ts", "latest"],
-    ["native", "js", "latest"]
+    ["web", "js", "8.7"]
+    // ["web", "ts", "latest"], //Disabling temporally while generator is not imported
```

Comment:
See my point above. I think we should make that `hot` work => this disable is not needed.

Reply:
Fixed